### PR TITLE
Clean up Get-SshPath test.

### PR DESCRIPTION
**Pull request by rkeithhill**
Also experimenting with how to mock git.exe and add some Get-GitStatus tests.

### CODE REVIEW

File: test/Get-GitStatus.Tests.ps1 [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+            $status.HasIndex | Should Be $false
+            $status.HasUntracked | Should Be $false
+            $status.Index.Count | Should Be 0
+            $status.Working.Count | Should Be 2
+            $status.Working[0] | Should Be "test/Foo.Tests.ps1"
+            $status.Working[1] | Should Be "test/Bar.Tests.ps1"
+            $status.AheadBy  | Should Be 0
+            $status.BehindBy | Should Be 0
+            $status.StashCount | Should Be 0
+            $status.Upstream -eq $null | Should Be $true
```
Comment (dahlbyk) [inline test/Get-GitStatus.Tests.ps1]: We can discuss more as additional tests are written, but I try to limit asserts to only what I'm specifically testing, e.g. `HasIndex`, `HasUntracked`, `Index` and `Working` for this set. A different set of tests would have asserts for `Branch`, `AheadBy`, `BehindBy`, `Upstream` (and `UpstreamGone`).

Comment (rkeithhill) [inline test/Get-GitStatus.Tests.ps1]: Fair enough.  I was a bit overexcited, getting the git.exe mocking to work.  :-)

Comment (rkeithhill) [inline test/Get-GitStatus.Tests.ps1]: Also, was in mode of testing everything.  Feel free to tweak the assertions at any time.

Comment (dahlbyk) [inline test/Get-GitStatus.Tests.ps1]: > I was a bit overexcited, getting the git.exe mocking to work. :-)

🎉  Understood 🎉 

> Feel free to tweak the assertions at any time.

I'm not worried about it today, unless you're planning to add more tests. What you have so far is good to merge as a starting point unless you'd rather keep going.

**Pull request merged**